oshc/oshc/settings.py

```diff
@@ -158,12 +158,9 @@
 STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
 
 STATICFILES_DIRS = (os.path.join(BASE_DIR, 'main/'), )
-
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_HOST_USER = 'email'
-EMAIL_HOST_PASSWORD = 'password'
-EMAIL_PORT = 587
-EMAIL_USE_TLS = True
```

Comment:
If you want to try please go ahead and merge to predev, we need to make sure necessary issues are open. refing https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/pull/222#pullrequestreview-87078476.

Reply:
We also have https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/202, we will keep this open and I feel fine with closing the mini ones, no probs.